crates/torii/core/src/processors/mod.rs

```diff
@@ -19,9 +19,9 @@ pub trait EventProcessor<P: Provider + Sync> {
         world: &WorldContractReader<'_, P>,
         db: &mut Sql,
         provider: &P,
-        block: &BlockWithTxs,
```

Comment:
hm why remove this? we might still want block context here

Reply:
Ah was trying to stay under clippy args limit, but yeah can add back in